### PR TITLE
Add trailing slash to gitlabPath and apiPath on save settings

### PR DESCRIPTION
**Pull request by da0shi**
Hi,

Links to commits were broken when I forgot to add trailing slash to GitLab Path & GitLab API Path settings.
So I've made little util to add trailing slash at the point user save their settings.

Hope it helps.

### CODE REVIEW

**Comment by sue445** [comment]
:+1:

File: src/util.js [hunk 1]
```diff
@@ -40,13 +40,20 @@ var util = (function(){
         return MD5_hexhash(JSON.stringify(obj));
     }
 
+    function addTrailingSlash(url){
+        if (typeof url !== "string") return url;
+        if (url === "" || url.match(/\/$/)) return url;
+        return url += '/';
```
Comment (sue445) [inline src/util.js]: I think `return url + '/' is better than `return url += '/'`

**Comment by da0shi** [comment]
Oh yes, you're right.
I fixed operator :+1:

**Comment by sue445** [comment]
LGTM!

**Comment by sue445** [comment]
@da0shi I released to Chrome webstore. Please wait.

**Comment by da0shi** [comment]
@sue445 I've got `1.1.2` now.  It works fine :+1: 
Thank you for accepting my pull request.

**Pull request merged**